### PR TITLE
Fix Link/Yink/Tink upB softlocks

### LINKED ISSUE #446: Link/tink/yink upB softlocks
Need to check status script

### PR DESCRIPTION
**Pull request by jobrien97**
removed infinite return loop

also removed redundant opff for the links

Fixes #446

### CODE REVIEW

File: fighters/younglink/src/status.rs [hunk 1]
```diff
@@ -137,26 +145,23 @@ unsafe extern "C" fn sub_specialhi_end_Main(fighter: &mut L2CFighterCommon) -> L
...
+                }
             }
-            if link_situation_helper(fighter).get_bool() && fighter.global_table[SITUATION_KIND] == SITUATION_KIND_GROUND {
-                fighter.change_status(
-                    L2CValue::I32(*FIGHTER_STATUS_KIND_LANDING),
-                    L2CValue::Bool(true)
-                );
-                return 1.into()
+            else {
+                if GroundModule::can_entry_cliff(fighter.module_accessor) == 1 && fighter.global_table[STICK_Y].get_f32() > -0.66 {
```
Comment (techyCoder81) [inline fighters/younglink/src/status.rs:158]: pretty sure you can just `fighter.stick_y()` now

File: fighters/link/src/status.rs [hunk 2]
```diff
@@ -258,6 +258,19 @@ pub unsafe fn pre_specialhi_end(fighter: &mut L2CFighterCommon) -> L2CValue {
 pub unsafe fn specialhi_end(fighter: &mut L2CFighterCommon) -> L2CValue {
     WorkModule::enable_transition_term(fighter.module_accessor, *FIGHTER_STATUS_TRANSITION_TERM_ID_FALL_SPECIAL);
     WorkModule::enable_transition_term(fighter.module_accessor, *FIGHTER_STATUS_TRANSITION_TERM_ID_WAIT);
+    if fighter.global_table[SITUATION_KIND] == SITUATION_KIND_GROUND {
```
Comment (techyCoder81) [inline fighters/link/src/status.rs:261]: this can just be `fighter.is_situation(*SITUATION_KIND_GROUND)` iirc

**Review by techyCoder81 — APPROVED** [review]
overall not really ecstatic that theres zero comments throughout this code

File: fighters/link/src/status.rs (hunk 2 above)
Comment (blu-dev) [reply fighters/link/src/status.rs:261]: it can

**Pull request merged**